config/application.rb

```diff
@@ -76,24 +76,37 @@ class Application < Rails::Application
           # necessarily need replicas.
           # Docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html
           raise "Elasticsearch status is #{status}" unless %w[green yellow].include?(status)
+
           "ok"
         end
 
-        ping.check :github do
-          uri = URI("https://status.github.com/api/status.json")
-          status = JSON.parse(Net::HTTP.get(uri))["status"]
-          raise "GitHub status is #{status}" unless status == "good"
+        ping.check :github, timeout: 5 do
+          uri = URI("https://www.githubstatus.com/api/v2/components.json")
+          status_components = JSON.parse(Net::HTTP.get(uri))["components"]
+          raise "GitHub status format is invalid!" if status_components.blank?
+
+          github_status_api_id = "brv1bkgrwx7q"
+          api_component = status_components.find do |component|
+            component["id"] == github_status_api_id
```

Comment:
What is the id `brv1bkgrwx7q`? Is this global to all integrators? Is this a secret?

Reply:
It is the `id` for the GitHub Status API component from https://www.githubstatus.com/api/v2/components.json.